lib/collection/src/collection_manager/holders/segment_holder.rs

```diff
@@ -523,6 +523,7 @@ impl<'s> SegmentHolder {
     /// Moving is not performed in the following cases:
     /// - The segment containing the point is appendable.
     /// - The `update_nonappendable` function returns true for the segment.
+    ///
```

Comment:
```
error: doc list item missing indentation
   --> lib/collection/src/collection_manager/holders/segment_holder.rs:526:9
    |
526 |     /// Otherwise, the operation is applied to the containing segment in place.
    |         ^
    |
    = help: if this is supposed to be its own paragraph, add a blank line
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#doc_lazy_continuation
    = note: `-D clippy::doc-lazy-continuation` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(clippy::doc_lazy_continuation)]`
help: indent this line
    |
526 |     ///   Otherwise, the operation is applied to the containing segment in place.
    |         ++

error: could not compile `collection` (lib) due to 1 previous error
```